apps/chat/src/components/Common/FolderContextMenu.tsx

```diff
@@ -146,7 +148,10 @@ export const FolderContextMenu = ({
         name: t('Unpublish'),
         dataQa: 'unpublish',
         display:
-          isPublishingEnabled && isItemPublic(folder.id) && !!onUnpublish,
+          isPublishingEnabled &&
+          isItemPublic(folder.id) &&
+          !!onUnpublish &&
+          isSidePanelFolder,
```

Comment:
use `featureType !== FeatureType.File` instead

Reply:
but we have `              
type === SharingType.ConversationFolder ? FeatureType.Chat : FeatureType.Prompt` in `ChangePtathDialog.tsx`